.github/copilot-instructions.md

```diff
@@ -44,35 +82,78 @@ In building and testing, never use `dotnet` without extension. Use `./dotnet.sh`
 
 ## Running tests
 
-(1) Build from the root with `build.sh`.
+(1) Build from the root with `./build.sh` (~3-5 minutes).
 (2) If that produces errors, fix those errors and build again. Repeat until the build is successful.
-(3) To then run tests, use a command similar to this `dotnet.sh test tests/Aspire.Seq.Tests/Aspire.Seq.Tests.csproj --no-build` (using the path to whatever projects are applicable to the change).
+(3) To run tests for a specific project: `dotnet test tests/ProjectName.Tests/ProjectName.Tests.csproj --no-build -- --filter-not-trait "quarantined=true"`
 
 Note that tests for a project can be executed without first building from the root.
 
-(4) To run just certain tests, it's important to include the filter after `--`, for example:
-
-dotnet.sh test tests/Aspire.Hosting.Testing.Tests/Aspire.Hosting.Testing.Tests.csproj --logger "console;verbosity=detailed" -- --filter-method "TestingBuilderHasAllPropertiesFromRealBuilder"
+(4) To run specific tests, include the filter after `--`:
+```bash
+dotnet test tests/Aspire.Hosting.Testing.Tests/Aspire.Hosting.Testing.Tests.csproj -- --filter-method "TestingBuilderHasAllPropertiesFromRealBuilder" --filter-not-trait "quarantined=true"
```

Comment:
is it clear enough that they should be using `dotnet.sh` from the root? there is a global dotnet installed on these machines (for MCP servers) -- they should not use that.

Reply:
Does paths have a fallback for the globally installed dotnet in case it is already present? In that case, our build won't have it under the .dotnet folder so we should make sure that works.